roles/configure-nexus/tasks/labs-repos.yml

```diff
@@ -23,7 +23,7 @@
 
 - name: Create New Script
   uri:
-    url: "http://{{ nexus_url }}/service/siesta/rest/v1/script/"
+    url: "https://{{ nexus_url }}/service/siesta/rest/v1/script/"
```

Comment:
None of these urls work because the default route to nexus is not secured. Please either make nexus template level changes to the route, or switch this back to http. This prevents the ansible automation to complete and needs manual intervention.